lambda/checkActivation/index.js

```diff
@@ -1,28 +1,33 @@
-const { endOfToday, compareAsc, addHours, startOfDay } = require('date-fns');
-const { utcToZonedTime, zonedTimeToUtc } = require('date-fns-tz');
+const { DateTime } = require('luxon');
 
 const { setStatus,
-        getPassesByStatus,
-        getParks,
-        getFacilities,
-        RESERVED_STATUS,
-        ACTIVE_STATUS,
-        EXPIRED_STATUS,
-        PASS_TYPE_PM,
-        timeZone } = require('../dynamoUtil');
+  getPassesByStatus,
+  getParks,
+  getFacilities,
+  RESERVED_STATUS,
+  ACTIVE_STATUS,
+  EXPIRED_STATUS,
+  PM_ACTIVATION_HOUR,
+  PASS_TYPE_PM,
+  TIMEZONE } = require('../dynamoUtil');
 const { sendResponse } = require('../responseUtil');
 
 exports.handler = async (event, context) => {
   console.log('Event:', event, context);
+  console.log('Server Time Zone:',
+    Intl.DateTimeFormat().resolvedOptions().timeZone || 'undefined',
+    `(${DateTime.now().toISO()})`
+  );
   try {
-    const theDate = zonedTimeToUtc(endOfToday(), timeZone);
+    const currentPSTDateTime = DateTime.now().setZone(TIMEZONE);
+    const endOfPSTDayUTCDateTime = currentPSTDateTime.endOf('day').toUTC();
```

Comment:
Does the job still need to run every hour if we are filtering on end-of-day?  i.e. would it just update 0 records for 23 out of 24 hourly runs?